packages/material-ui/src/utils/useId.js

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+/**
+ * Private module reserved for @material-ui/x packages.
+ */
+export default function useId(idProp, skipDoubleRender) {
```

Comment:
Not a big fan of `skipDoubleRender` terminology, looking for suggestions.

Reply:
What about we remove this argument? All the tests are green without.

Reply:
This argument originally came from `Tooltip.js` that would skip `setState` if it was `!open`, so I just wanted to be safe and preserved it. But I'm happy to drop it? Are you okay with `Tooltip` running the render phase twice?

Reply:
I'm happy to drop it - 1 less thing to write a test for 😉

Reply:
We have no benchmark to measure the difference